userspace/libscap/scap_open.h

```diff
@@ -31,80 +31,63 @@ limitations under the License.
 #include "../../driver/ppm_events_public.h"
 
 #ifdef __cplusplus
-extern "C" {
+extern "C"
+{
 #endif
 
-/*!
-  \brief Arguments for scap_open
-*/
-typedef enum {
-	/*!
-	 * Default value that mostly exists so that sinsp can have a valid value
-	 * before it is initialized.
-	 */
-	SCAP_MODE_NONE = 0,
-	/*!
-	 * Read system call data from a capture file.
-	 */
-	SCAP_MODE_CAPTURE,
-	/*!
-	 * Read system call data from the underlying operating system.
-	 */
-	SCAP_MODE_LIVE,
-	/*!
-	 * Do not read system call data. If next is called, a dummy event is
-	 * returned.
-	 */
-	SCAP_MODE_NODRIVER,
 	/*!
-	 * Do not read system call data. Events come from the configured input plugin.
-	 */
-	SCAP_MODE_PLUGIN,
-#ifdef HAS_ENGINE_MODERN_BPF
+	  \brief Scap possible modes
+	*/
+	typedef enum
+	{
+		/*!
+		 * Default value that mostly exists so that sinsp can have a valid value
+		 * before it is initialized.
+		 */
+		SCAP_MODE_NONE = 0,
+		/*!
+		 * Read system call data from a capture file.
+		 */
+		SCAP_MODE_CAPTURE,
+		/*!
+		 * Read system call data from the underlying operating system.
+		 */
+		SCAP_MODE_LIVE,
+		/*!
+		 * Do not read system call data. If next is called, a dummy event is
+		 * returned.
+		 */
+		SCAP_MODE_NODRIVER,
+		/*!
+		 * Do not read system call data. Events come from the configured input plugin.
+		 */
+		SCAP_MODE_PLUGIN,
```

Comment:
Isn't this now a case of `SCAP_MODE_LIVE` with `scap->engine == sourceplugin`  (or whatever it is called :D )